aiotractive/api.py

```diff
@@ -75,6 +79,11 @@ async def raw_request(self, uri, params=None, data=None, method="GET"):
             headers=await self.auth_headers(),
             timeout=self._timeout,
         ) as response:
+            _LOGGER.debug("Request %s, status: %s", response.url, response.status)
+            if response.status == 429:
```

Comment:
The general approach for retries is usually a randomized exponential delay. For instance let's say we have 3 retry attempts:
1. 1s + rand(n)
2. 4s + rand(n)
3. 9s + rand(n)

The retry count and a function that calculates the delay can be passed to `API` from the outside, :
```python
api = Api("login", "password", retry_count=3, retry_delay=lambda a: a**2 + random.uniform(0, 3))
```
`a` is an attempt number, counts from 1.

WDYT?